<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OrbitMainWindow</class>
 <widget class="QMainWindow" name="OrbitMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>976</width>
    <height>594</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Orbit Profiler</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter_2">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTabWidget" name="MainTabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>home</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_7">
         <item row="0" column="1">
          <widget class="QSplitter" name="HomeHorizontalSplitter">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <widget class="QSplitter" name="HomeVerticalSplitter">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="baseSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <widget class="ProcessLauncherWidget" name="ProcessesList" native="true"/>
            <widget class="OrbitDataViewPanel" name="SessionList"/>
           </widget>
           <widget class="OrbitDataViewPanel" name="ModulesList"/>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="DebugTab">
        <attribute name="title">
         <string>debug</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="OrbitGLWidget" name="DebugGLWidget"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="CaptureTab">
        <attribute name="title">
         <string>capture</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="0">
          <widget class="OrbitGLWidget" name="CaptureGLWidget"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="OutputTab">
        <attribute name="title">
         <string>output</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_13">
         <item row="0" column="0">
          <widget class="OrbitCodeEditor" name="plainTextEdit"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="VisualizeTab">
        <attribute name="title">
         <string>visualize</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="0" column="0">
          <widget class="OrbitGLWidget" name="VisualizeGLWidget"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="WatchTab">
        <attribute name="title">
         <string>watch</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_14">
         <item row="0" column="0">
          <widget class="OrbitWatchWidget" name="WatchWidget" native="true"/>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QTabWidget" name="RightTabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="FunctionsTab">
        <attribute name="title">
         <string>functions</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="OrbitDataViewPanel" name="FunctionsList"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="TypesTab">
        <attribute name="title">
         <string>types</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_11">
         <item row="0" column="0">
          <widget class="OrbitDataViewPanel" name="TypesList"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="GlobalsType">
        <attribute name="title">
         <string>globals</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_12">
         <item row="0" column="1">
          <widget class="OrbitDataViewPanel" name="GlobalsList"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="LiveTab">
        <attribute name="title">
         <string>live</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_15">
         <item row="0" column="0">
          <widget class="OrbitDataViewPanel" name="LiveFunctionsList"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="CallStackTab">
        <attribute name="title">
         <string>callstack</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_8">
         <item row="0" column="0">
          <widget class="OrbitDataViewPanel" name="CallStackView"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="CodeTab">
        <attribute name="title">
         <string>code</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_10">
         <item row="0" column="0">
          <widget class="QLineEdit" name="lineEdit"/>
         </item>
         <item row="1" column="0">
          <widget class="OrbitCodeEditor" name="CodeTextEdit">
           <property name="accessibleName">
            <string>CodeTextEdit</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QWidget" name="FileMappingWidget" native="true">
           <layout class="QGridLayout" name="gridLayout_9">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item row="0" column="0">
             <widget class="QPushButton" name="SaveFileMapping">
              <property name="text">
               <string>Save File Mapping</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="OrbitCodeEditor" name="FileMappingTextEdit">
              <property name="accessibleName">
               <string>FileMappingTextEdit</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="outputTab">
        <attribute name="title">
         <string>debug</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <widget class="OrbitDataViewPanel" name="OutputView"/>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>976</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave_Capture"/>
    <addaction name="actionOpen_Capture_2"/>
    <addaction name="separator"/>
    <addaction name="actionSave_Session"/>
    <addaction name="actionSave_Session_As"/>
    <addaction name="actionOpen_Session"/>
    <addaction name="actionOpen_PDB"/>
    <addaction name="separator"/>
    <addaction name="actionLaunch_Process"/>
    <addaction name="actionDisconnect"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuDev">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionEnable_Sampling"/>
    <addaction name="actionEnable_Context_Switches"/>
    <addaction name="actionEnable_Unreal_Support"/>
    <addaction name="actionAllow_Unsafe_Hooking"/>
    <addaction name="actionOutputDebugString"/>
    <addaction name="actionUploadDumpsToServer"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionShow_Includes_Util"/>
    <addaction name="actionDiff"/>
    <addaction name="actionRule_Editor"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuDev"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionSave_Session">
   <property name="text">
    <string>Save Session</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionOpen_Session">
   <property name="text">
    <string>Open Session</string>
   </property>
  </action>
  <action name="actionOpen_PDB">
   <property name="text">
    <string>Open PDB</string>
   </property>
  </action>
  <action name="actionOpen_Capture">
   <property name="text">
    <string>Open Capture</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="text">
    <string>Disconnect</string>
   </property>
  </action>
  <action name="actionStart_Captue">
   <property name="text">
    <string>Start Captue</string>
   </property>
  </action>
  <action name="actionStop_Capture">
   <property name="text">
    <string>Stop Capture</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionToogleDevMode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Dev Mode</string>
   </property>
  </action>
  <action name="actionSave_Session_As">
   <property name="text">
    <string>Save Session As...</string>
   </property>
  </action>
  <action name="actionEnable_Context_Switches">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Context Switches</string>
   </property>
  </action>
  <action name="actionLaunch_Process">
   <property name="text">
    <string>Launch Process</string>
   </property>
  </action>
  <action name="actionEnable_Unreal_Support">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Unreal Support</string>
   </property>
  </action>
  <action name="actionAllow_Unsafe_Hooking">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Unsafe Hooking</string>
   </property>
  </action>
  <action name="actionEnable_Sampling">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Sampling</string>
   </property>
  </action>
  <action name="actionOpen_Capture_2">
   <property name="text">
    <string>Open Capture</string>
   </property>
  </action>
  <action name="actionSave_Capture">
   <property name="text">
    <string>Save Capture</string>
   </property>
  </action>
  <action name="actionShow_Includes_Util">
   <property name="text">
    <string>Show Includes Util</string>
   </property>
  </action>
  <action name="actionDiff">
   <property name="text">
    <string>Diff</string>
   </property>
  </action>
  <action name="actionOutputDebugString">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>OutputDebugString</string>
   </property>
  </action>
  <action name="actionUploadDumpsToServer">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Upload Dumps on Crash</string>
   </property>
  </action>
  <action name="actionRule_Editor">
   <property name="text">
    <string>Rule Editor</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>OrbitCodeEditor</class>
   <extends>QPlainTextEdit</extends>
   <header>orbitcodeeditor.h</header>
  </customwidget>
  <customwidget>
   <class>OrbitGLWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>orbitglwidget.h</header>
  </customwidget>
  <customwidget>
   <class>OrbitDataViewPanel</class>
   <extends>QTreeView</extends>
   <header>orbitdataviewpanel.h</header>
  </customwidget>
  <customwidget>
   <class>ProcessLauncherWidget</class>
   <extends>QWidget</extends>
   <header>processlauncherwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>OrbitWatchWidget</class>
   <extends>QWidget</extends>
   <header>orbitwatchwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
